packages/ListBox/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
-- Moved from Listbox
+- Moved from ListBox
```

Comment:
Cool... but what about your _real_ changes you made in this PR? 😉

Reply:
lool true!